lib/Runtime/Language/SimpleDataCacheWrapper.h

```diff
@@ -69,8 +69,13 @@ namespace Js
 #ifdef ENABLE_WININET_PROFILE_DATA_CACHE
             ULONG bytesWritten = 0;
             hr = this->outStream->Write(&data, sizeof(T), &bytesWritten);
+            Assert(bytesWritten == sizeof(T) || hr == S_FALSE);
```

Comment:
Assert should check for failed HR (also in other places)

Reply:
Good point, I've updated it.